src/test/java/moduletwo4/hardcore/test/CloudGoogleTest.java

```diff
@@ -0,0 +1,81 @@
+package moduletwo4.hardcore.test;
+
+import moduletwo4.hardcore.page.EmailYourEstimatePage;
+import moduletwo4.hardcore.page.MinuteMailPage;
+import moduletwo4.hurtmeplenty.page.CloudGoogleCalculatorPage;
+import moduletwo4.hurtmeplenty.page.CloudGooglePage;
+import moduletwo4.hurtmeplenty.page.ResultsPastebinPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CloudGoogleTest {
+
+    private static final String SEARCH_LINE = "Google Cloud Platform Pricing Calculator";
+    private static CloudGoogleCalculatorPage calculatorPage;
+    private static ResultsPastebinPage resultsPage;
+    private static int WAIT_TIMEOUT_SECONDS = 60;
+    private static EmailYourEstimatePage emailPage;
+    private static MinuteMailPage minuteMailPage;
+    private WebDriver driver;
+    private ArrayList<String> tabs;
+    private int firstTab = 0;
+    private int secondTab = 1;
+
+
+    @BeforeClass
+    public void getPrecondition() {
+        driver = new FirefoxDriver();
```

Comment:
What a point in using another browser? As the binary is missing too - this test fails for me. And I'm too lazy to download another binary ))